### ISSUE TITLE
Are there plans to remove the clashes with inbox modules like Microsoft.PowerShell.Archive in v5?

### ISSUE THREAD

**Issue by kilasuit** [Pscx/Pscx#8]
As we are getting more and more functionality coming 'inbox' I think that there would be benefits to making some changes to PSCX that will not enforce differences in the command preferences between modules like Microsoft.PowerShell.Archive.

The simplest suggestion would be to prefix commands like 

``` powershell
Expand-Archive
```

 to be more like

``` powershell
 Expand-PSCXArchive 
```

I know that this isn't ideal but now that we have Expand-Archive inbox (& OS too) then perhaps the PSCX version could end up being retired and the differences merged into Microsoft.PowerShell.Archive ?

This may be a discussion that becomes much wider than PSCX so it would be good to get your thoughts on this @rkeithhill

**Comment by rkeithhill** [comment]
You can control the prefix today with `ipmo pscx -prefix pscx`.  However I prefer using `Pscx\Expand-Archive`.  

We have been retiring commands as PowerShell gets built-in equivalents (Start-Process, Select-Xml, Get-Random, Unblock-File, Get-AlternateDataStream).  Unfortunately, I don't think the built-in Compress/Expand-Archive aren't quite equivalent yet.  But yeah, the right answer long term is to contribute the implementation of those differences to the MS.PS.Archive repo.

Another example of a clash that I don't want to remove just yet is Format-Hex.  The built-in command doesn't support Count and Offset parameters.  I use Count all the time to peek at the beginning of a file to see if it has a BOM or not.  BTW in the upcoming 3.2.2. drop, we will alias fhex to PSCX\Format-Hex (instead of the built-in command).  That has been driving me nuts.  :-)
